beacon-chain/blockchain/chain_info.go

```diff
@@ -94,47 +92,27 @@ type OptimisticModeFetcher interface {
 }
 
 // FinalizedCheckpt returns the latest finalized checkpoint from chain store.
-func (s *Service) FinalizedCheckpt() (*ethpb.Checkpoint, error) {
-	cp, err := s.store.FinalizedCheckpt()
-	if err != nil {
-		return nil, err
-	}
-
-	return ethpb.CopyCheckpoint(cp), nil
+func (s *Service) FinalizedCheckpt() *ethpb.Checkpoint {
+	cp := s.ForkChoicer().FinalizedCheckpoint()
+	return &ethpb.Checkpoint{Epoch: cp.Epoch, Root: bytesutil.SafeCopyBytes(cp.Root[:])}
```

Comment:
because `ForkChoicer` returns a `forkchoicetypes.Checkpoint` not an `ethpb.Checkpoint`, we'll move eventually to the former and then we'll add that helper back.